students/lauraannf/lessons/lesson04/assignment/customer_model.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Apr  1 15:41:41 2019
+
+@author: Laura.Fiorentino
+
+Lesson3 Assignment
+"""
+
+
+from peewee import *
```

Comment:
This is also a good opportunity for improving your code. Try to avoid wildcard imports, in particular with large modules such as Peewee, you will end up, like in this case, with a lot of unused imports. Instead, import only those methods you will be using.